apps/nextra/pages/en/build/smart-contracts/book/variables.mdx

```diff
@@ -763,11 +763,12 @@ script {
   fun example() {
     {
       let x = 0;
-      x + 1; // value is discarded
+      x + 1; // 
```

Comment:
This value is discarded.

Because `x + 1` isn't set to anything

Reply:
The truth is, it was an accident.